clustoapi/apps/attribute.py

```diff
@@ -21,6 +21,61 @@
 app.config['source_module'] = __name__
 
 
+def _write_attrs(method, name, **kwargs):
+    """
+Helper method for reduced code between POST and PUT.
+Returns a response for the methods calling it.
+"""
+    if method == 'set_attr':
+        code = 200
+    if method == 'add_attr':
+        code = 201
+    else:
```

Comment:
I kinda wanna have the check at the top of the method, i.e. "if method not in () then bail"